### DISCUSSION TITLE
Fork a project to an existing Repository. Is it possible?

### DISCUSSION THREAD

**Discussion by PuffCab** [community/community#67713] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I am relatively new to github. 
I have to to fork different repositories, from different github accounts, do some changes and then do a pull request. 
Everytime I fork a project, it creates a new repository in my list of repositories. 
I would like to keep them more organised, by having all those forked repos inside the same repository in my account. 

The idea would be to create a "forked_repos" repository, and every time I fork one, being able to select that destiny
Is there a way to do that? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by spenserblack** [comment, marked as answer]
# Short answer

No, I don't think so. Long-term, it's probably best to stick with 1 repository per "fork."

# Long answer

Yes, but it won't be easy.

You can partially do it like so:
```shell
# in forked_repos
git remote add repo-to-fork https://github.com/owner/repo-to-fork.git
git fetch repo-to-fork
```

Now you have the history of `repo-to-fork` available in your forks repo.

```shell
git branch repo-to-fork/main my-contribution
git checkout my-contribution
git push -u origin my-contribution
```

Now you've created a branch called `my-contribution` off of the `repo-to-fork`'s `main` branch, and set git up so that you push to the `my-contribution` branch in your remote `forked_repos` repository.

However, once you've done that, AFAIK GitHub will not allow you to create pull requests, because GitHub doesn't know that you've "forked" the repository. GitHub will think that they're unrelated repositories. So you'll only be able to contribute if the maintainers take contributions *other than pull requests,* like emailing a patch.

# Alternative Solutions

Some users create organizations to put their forks, like https://github.com/cutlery-drawer. Personally, I just delete forks once my PR has been merged, unless I think I'll continue to contribute in the future.

**Reply by PuffCab** [reply]
Creating an organisation sound like a practical and simple way. The first one is a bit above my skills right know, and not being able to do a pull request afterwards, would be a problem . 
The second though, that is a very good idea :). 
Thanks a lot!

**Reply by jaikumar112** [reply]
Hello GitHub Support Team,

I am unable to login to my GitHub account @Jaikumar110.

Problem:
- My 2FA authenticator app is not generating codes
- I do not have access to my recovery codes
- I am unable to sign in to my account

Account details:
- Username: @Jaikumar110
- Email: jairamofficail@gmail.com

I am the original owner of this account. Please help me recover access or verify my identity so I can log back in.

Thank you,
Jaikumar

**Reply by johanserkar-lgtm** [reply]
I have a bate sid new

**Comment by RumeysaaKopuz** [comment]
of course you can

**Reply by PuffCab** [reply]
A bit more elaborated reply would be much appreciated ;)

**Comment by RumeysaaKopuz** [comment]
absoultly #2

**Comment by Charlsgg** [comment]
GitHub does not provide a direct built-in feature to fork a project into an existing repository. When you fork a repository on GitHub, it creates a new repository under your account. However, you can achieve a similar outcome by manually copying the contents of the forked repository into your existing repository.

**Comment by Kostikyan** [comment]
absoultly #2

**Comment by Benedek-Labancz** [comment]
As an alternative, you can clone a repository as submodule (a Git repository as a subdirectory of another Git repository) by running
`git submodule add <url> <folder-name>`

I am not sure if this is the behavior you are looking for, please refer to the documentation for more details: 
https://git-scm.com/book/en/v2/Git-Tools-Submodules

**Comment by GitHu39** [comment]
![17471314502613526896857440961071](https://github.com/user-attachments/assets/27a9f883-2dc9-44d1-a939-e82c76ff0016)

**Reply by oomodan** [reply]
git branch repo-to-fork/main my-contribution
git checkout my-contribution
git push -u origin my-contribution

**Comment by GitHu39** [comment]
Thank for. Your requirement

**Comment by babulalisarkar99-bit** [comment]
96wingo.online

**Comment by babulalisarkar99-bit** [comment]
96wingo.online

**Comment by babulalisarkar99-bit** [comment]
93659 21484

**Comment by babulalisarkar99-bit** [comment]
Rahul 123456

**Comment by babulalisarkar99-bit** [comment]
AIRP000001

**Comment by babulalisarkar99-bit** [comment]
Airtel payments Bank

**Comment by babulalisarkar99-bit** [comment]
3000

**Comment by jaikumar112** [comment]
Hello GitHub Support Team,

I am unable to login to my GitHub account @Jaikumar110.

Problem:
- My 2FA authenticator app is not generating codes
- I do not have access to my recovery codes
- I am unable to sign in to my account

Account details:
- Username: @Jaikumar110
- Email: jairamofficail@gmail.com

I am the original owner of this account. Please help me recover access or verify my identity so I can log back in.

Thank you,
Jaikumar

**Comment by khansarwar51296-design** [comment]
> ### Select Topic Area
> Question
> 
> ### Feature Area
> Issues
> 
> ### Body
> I am relatively new to github. I have to to fork different repositories, from different github accounts, do some changes and then do a pull request. Everytime I fork a project, it creates a new repository in my list of repositories. I would like to keep them more organised, by having all those forked repos inside the same repository in my account.
> 
> The idea would be to create a "forked_repos" repository, and every time I fork one, being able to select that destiny Is there a way to do that?
> 
> ### Guidelines
> * [x]  I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

[LF-Projects-Hosted-Project-Tools-Project-Review-Form.pdf](https://github.com/user-attachments/files/28025144/LF-Projects-Hosted-Project-Tools-Project-Review-Form.pdf)

**Comment by eli197983-hue** [comment]
-

**Comment by Safwanni2212** [comment]
Com WhatsApp Messenger
+923156237453

safwani112
